koala/ast/graph.py

```diff
@@ -1023,6 +1022,18 @@ def make_subgraph(G, seed, direction = "ascending"):
 
     return subgraph
 
+def cell2code(named_ranges, cell, sheet):
```

Comment:
One day we should tidy up all the mess and put these functions in a `graph_utils.py`